asciidoc/modules/con_mg_migrating-authorization-applications.adoc

```diff
@@ -0,0 +1,39 @@
+[id="migrating-authorization-applications_{context}"]
+= Migrating the authorization applications
+
+Authorization is a new feature in {VertX} {4}. In earlier releases, you could only check if a user was authorized to perform the tasks on the `User` object. This meant that the provider was responsible for both authentication and authorization of the user. In {VertX} {4}, the user does not know who is the provider, so it can be used on different providers.
```

Comment:
@vietj What does "it" mean in the following sentence?
"In {VertX} {4}, the user does not know who is the provider, so **it** can be used on different providers."

Reply:
@pmlopes can you address this question ?

Reply:
Hi @sangeetaraghu, I wanted to say:

In {VertX} {4} the `User` object instances are not coupled to a authentication provider, which means that now, for example, you can authenticate a user using `OAuth2` and perform authorization checks against a Mongo or SQL database.